### PR TITLE
(READY) Adds Keratin Plates genetic mutation

### PR DESCRIPTION
**Pull request by Arzion5**
:cl:
add: Added Keratin Plates, a genetic superpower that reduces damage done to you by a little bit. Good enough to yawn punches off but not good enough to protect against armor-piercing attacks or much stronger weapons effectively.
/:cl:

Genetics variety, this is the first defense-oriented genetic power. Added mostly for fun value.

Things to do : 

- [x] Make the code actually work

### CODE REVIEW

**Review by ghost — CHANGES_REQUESTED** [review]
You WILL need to change a /tg/ file for this --- change DNA_STRUC_ENZYMES_BLOCKS to 20 in __DEFINES/DNA.dm

**Comment by tortellinitony** [comment]
no you can just redefine it

**Comment by Arzion5** [comment]
Maybe later. I will do it in a respectable time-frame

**Comment by KayeArray** [comment]
+1

**Comment by ghost** [comment]
sounds cool
+1

File: hippiestation/code/datums/mutations.dm [hunk 1]
```diff
@@ -56,6 +56,26 @@
...
+	if(..())
+		return
+	var/datum/species/S = owner.dna.species
+	S.armor = 5 // You're resistant to punches.
+
+/datum/mutation/human/steel/on_losing(mob/living/carbon/human/owner)
+	if(..())
+		return
+	var/datum/species/S = owner.dna.species
+	S.armor = 0 //You're no longer resistant to punches. Do not combine with DNA Vault.
```
Comment (ghost) [inline hippiestation/code/datums/mutations.dm]: S.armor = initial(S.armor)

File: hippiestation/code/datums/mutations.dm [hunk 2]
```diff
@@ -56,6 +56,26 @@
...
+	get_chance = 15
+	lowest_value = 256 * 12
+	text_gain_indication = "<span class='notice'>You feel tougher.</span>"
+	text_lose_indication = "<span class='notice'>You feel weaker.</span>"
+
+/datum/mutation/human/steel/on_acquiring(mob/living/carbon/human/owner)
+	if(..())
+		return
+	var/datum/species/S = owner.dna.species
+	S.armor = 5 // You're resistant to punches.
```
Comment (ghost) [inline hippiestation/code/datums/mutations.dm]: S.armor = initial(S.armor) + 5

Comment (KayeArray) [reply hippiestation/code/datums/mutations.dm]: No, do S.armor += 5

**Comment by ghost** [comment]
rename to Subdermal Matrix/Keratin Skin/Light Carapace/Severe Psoriasis/Tough Epidermis/Bone Plates etc.  Man of steel sounds like shit tbh if it's not actually steel.

File: hippiestation/code/datums/mutations.dm [hunk 3]
```diff
@@ -56,6 +56,26 @@
...
+	if(..())
+		return
+	var/datum/species/S = owner.dna.species
+	S.armor += 5 // You're resistant to punches.
+
+/datum/mutation/human/keratin/on_losing(mob/living/carbon/human/owner)
+	if(..())
+		return
+	var/datum/species/S = owner.dna.species
+	S.armor = initial(S.armor) //You're no longer resistant to punches. Do not combine with DNA Vault.
```
Comment (KayeArray) [inline hippiestation/code/datums/mutations.dm]: S.armor -= 5

Now it won't remove DNA vault armour

**Comment by WallaceShark** [comment]
Cool concept! +1

**Comment by 2Dollarbill** [comment]
Interesting concept, honestly more balanced than a hulk mutation but that's the fun of hulk.
+1

**Comment by ghost** [comment]
Your PR needs to be updated to reflect the current state of the master branch, please update your PR and either (re)open it, or ask a maintainer to (re)open it.

**Pull request closed without merging**